example/notebooks/pneumonia.ipynb

```diff
@@ -0,0 +1,1083 @@
+    "                          target_names,\n",
+    "                          title='Confusion matrix',\n",
+    "                          cmap=None,\n",
+    "                          normalize=True):\n",
+    "    accuracy = np.trace(cm) / float(np.sum(cm))\n",
+    "    misclass = 1 - accuracy\n",
+    "\n",
+    "    if cmap is None:\n",
+    "        cmap = plt.get_cmap('Blues')\n",
+    "\n",
+    "    plt.figure(figsize=(8, 6))\n",
+    "    plt.imshow(cm, interpolation='nearest', cmap=cmap)\n",
+    "    plt.title(title)\n",
+    "    plt.colorbar()\n",
+    "\n",
+    "    if target_names is not None:\n",
+    "        tick_marks = np.arange(len(target_names))\n",
+    "        plt.xticks(tick_marks, target_names, rotation=45)\n",
+    "        plt.yticks(tick_marks, target_names)\n",
+    "\n",
+    "    if normalize:\n",
+    "        cm = cm.astype('float') / cm.sum(axis=1)[:, np.newaxis]\n",
+    "\n",
+    "\n",
+    "    thresh = cm.max() / 1.5 if normalize else cm.max() / 2\n",
+    "    for i, j in itertools.product(range(cm.shape[0]), range(cm.shape[1])):\n",
+    "        if normalize:\n",
+    "            plt.text(j, i, \"{:0.4f}\".format(cm[i, j]),\n",
+    "                     horizontalalignment=\"center\",\n",
+    "                     color=\"white\" if cm[i, j] > thresh else \"black\")\n",
+    "        else:\n",
+    "            plt.text(j, i, \"{:,}\".format(cm[i, j]),\n",
+    "                     horizontalalignment=\"center\",\n",
+    "                     color=\"white\" if cm[i, j] > thresh else \"black\")\n",
+    "\n",
+    "\n",
+    "    plt.tight_layout()\n",
+    "    plt.ylabel('True label')\n",
+    "    plt.xlabel('Predicted label\\naccuracy={:0.4f}; misclass={:0.4f}'.format(accuracy, misclass))\n",
+    "    plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.metrics import confusion_matrix\n",
+    "\n",
+    "cm =  confusion_matrix(y_true_list, y_pred_list)\n",
+    "\n",
+    "plot_confusion_matrix(cm = cm, \n",
+    "                      normalize    = False,\n",
+    "                      target_names = ['pneumonia','normal'],\n",
+    "                      title        = \"Confusion Matrix\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Testset results:**\n",
+    "- VGG19 93% test\n",
+    "- VGG11: 93,5%\n",
+    "- Resnet34: 93%\n",
+    "- Resnet18: 94%\n",
+    "- Resnet50: 94,25%\n",
+    "- Mobilenetv2: 92,25%"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Model Evaluation\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Testset Creation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "shutil.rmtree('./COVID19-DATASET/test/covid19')\n",
```

Comment:
Same here. You have to check if the directory exists.

Reply:
fixed